README.md

```diff
@@ -173,3 +173,40 @@ When enabled, this module will
 * initialize the state variable `target` to what is provided in the
   config file
 * print a message on every time step
+
+# Visualization Config
+
+To visualize the output of the simulation, please add the variable names to the list `visual_variables` in the cofig file `config.ini` under the `[visualization]` section
+```
+[visualization]
+# list of variables to visualize [variable_name|vtk_type]
+# vtk_type: STRUCTURED_POINTS, STRUCTURED_GRID, RECTILINEAR_GRID, UNSTRUCTURED_GRID, POLY_DATA
+visual_variables =  [
+                        {
+                            "module":"afumigatus",
+                            "variable":"tree",
+                            "vtk_type":"POLY_DATA",
+                            "attributes":["iron_pool", "iteration"]
+                        },
+                        {
+                            "module":"geometry",
+                            "variable":"lung_tissue",
+                            "vtk_type":"STRUCTURED_POINTS",
+                            "attributes":[]
+                        }
+                    ]
```

Comment:
The need for nested datastructures like this makes me wonder if we should go with toml or yaml over ini.  Maybe something to bring up in our next meeting.

Reply:
Yes. I agree that yaml would be a better config format to improve the flexibility for nested data, but I suggest we create another branch for this new feature. For this PR, I probably just keep this temporary raw JSON structure.

Reply:
Yes, that should happen in a different branch.